ajax/libs/weld/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "weld",
+  "version": "0.2.3",
+  "description": "Template antimatter for JavaScript",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/tmpvar/weld.git"
+  },
+  "author": "hij1nx <hij1nx@gmail.com>",
+  "license": "MIT",
+  "filename": "weld.min.js",
```

Comment:
Could you move this line to the second line? It's simpler to see and be revised if needed.